monitor/dbt_runner.py

```diff
@@ -51,12 +51,21 @@ def snapshot(self) -> bool:
         success, _ = self._run_command(['snapshot'])
         return success
 
-    def run_operation(self, macro_name: str, json_logs: bool = True, macro_args: dict = None,
-                      log_errors: bool = False) -> list:
+    def run_operation(
+        self,
+        macro_name: str,
+        json_logs: bool = True,
+        macro_args: Optional[dict] = None,
+        log_errors: bool = False,
+        dbt_vars: Optional[dict] = None
```

Comment:
cosmetics -  I'd rename dbt_vars to vars as it's already inside DbtRunner so no need for the prefix dbt here